src/import/converters.rs

```diff
@@ -337,9 +337,12 @@ fn conv_gradient_colors(
                 stops.push(
                     (
                         values[0] as f32,
-                        runtime::model::fixed::Color::rgba(
-                            values[1], values[2], values[3], values[4],
-                        ),
+                        runtime::model::fixed::Color::new([
+                            values[1] as f32,
+                            values[2] as f32,
+                            values[3] as f32,
+                            values[4] as f32,
+                        ]),
```

Comment:
What was the range on these values before?  Was it 0.0 to 1.0 or 0 to 255? If it is 0 to 255, then we'll have to divide them here as well.

Reply:
(Alternatively, use `Color::from_rgba()`?)

Reply:
The old Color::rgba method took normalized colors

https://docs.rs/peniko/0.2.0/peniko/struct.Color.html#method.rgba

Logic is correct


> (Alternatively, use `Color::from_rgba()`?)

No such method exists in the new color crate